src/test/scala/uk/gov/hmrc/test/ui/pages/registration/AmlActivityPage.scala

```diff
@@ -32,20 +33,24 @@ object AmlSupervisorPage extends BasePage {
     this
   }
 
-  def provideGcOrFcaAmlSupervisor(value: String): this.type = {
-    SharedActions.selectLabelByPartialText(value)
-    submitPage()
+  def submitPage(): this.type = {
+    SharedActions.clickButton()
     this
   }
 
-  def selectOtherWithNoProfessionalBodyAndSubmit: this.type = {
-    SharedActions.selectLabelByPartialText("Other")
-    submitPage()
-  }
-
-  def submitPage(): this.type = {
-    SharedActions.clickButton()
+  def selectYesOrNoAmlActivity(value: String): this.type = {
+    value match {
+      case "Yes" =>
+        SharedActions.selectLabelByPartialText("Yes")
+      case _     =>
+        SharedActions.selectLabelByPartialText("No")
+    }
```

Comment:
Given this is just selecting a 'Yes' or 'No' - we can use this everywhere else, so it can be a more generic shared action and call it just `selectYesOrNo`